src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +30,51 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_True_XV_15() {
+    Integer i = Integer.valueOf(15);
+    assertTrue(i.equals(RomanNumeral.convertFromString("XV")));
+  }
+
+  @Test
+  public void equals_False_XV_10() {
+    Integer i = Integer.valueOf(10);
+    assertFalse(i.equals(RomanNumeral.convertFromString("XX")));
+  }
+
+  @Test
+  public void equals_True_CI_101() {
+    Integer i = Integer.valueOf(101);
+    assertTrue(i.equals(RomanNumeral.convertFromString("CI")));
+  }
+
+  @ParameterizedTest
+  @CsvSource({
+    "V, 5",
+    "VI, 6",
+    "DCLXVI, 666",
+    "IV, 4",
+    "XCIX, 99",
+    "CDXCIX, 499",
+    "LV, 55",
+    "CMXLIV, 944",
+    "M, 1000",
+    "C, 100",
+    "DCC, 700",
+    "XI, 11",
+    "LXX, 70",
+    "VII, 7",
+    "VIIi, 8",
+    "LX, 60",
+    "LXX, 70",
+    "LXXX, 80",
+    "DC, 600",
+    "DCC, 700",
+    "DCCC, 800"
+  })
+  void convertFromStringReturnsCorrectInt(String input, Integer expected) {
+    int actual = RomanNumeral.convertFromString(input);
+    assertTrue(expected.equals(actual));
```

Comment:
This calls `Integer.equals()`. We want to test `RomanNumeral.equals()`.

Reply:
Ah, thanks for the clarification. I don't think this code is on my branch anymore.